cmd/config.go

```diff
@@ -51,17 +51,17 @@ PLEASE PLEASE PLEASE NEVER SHARE YOUR PRIVATE KEYS WITH ANYONE. EVER.
 Anyone with your private keys can impersonate you on qri.`,
 		Args: cobra.MaximumNArgs(1),
```

Comment:
Don't need to get to this in this PR, but I'd love to start moving away from using the `Args` field completely.